src/app/pages/ArticlePage/ArticlePage.tsx

```diff
@@ -227,6 +239,8 @@ const ArticlePage = ({ pageData }: { pageData: Article }) => {
     showRelatedTopics && topics.length > 0 && !isTransliterated,
   );
 
+  useOptimizelyScrollDepth();
```

Comment:
We'll need to keep this in the actual JumpTo component.

I think you'll just need to move the `useOptimizelyScrollDepth()` mock out of the ArticlePage test and into the JumpTo test to fix the previous test failures

Reply:
Ah okay 💡 thank you I'll change this.